src/main/resources/synthetic.xml

```diff
@@ -77,6 +139,8 @@
         <property name="wpAdminUsername" label="Administrative user" description="Username of the administrative user." category="Portal" required="false"/>
         <property name="wpAdminPassword" label="Administrative password" description="Password of the administrative user." category="Portal" required="false" password="true"/>
         <property name="wpConfigUrl" label="Configuration URL" description="The URL of the WebSphere Portal configuration API." category="Portal" required="false"/>
+        <property name="portalHost" kind="ci" referenced-type="overthere.Host" label="Portal Host" description="The host xml access is on." category="Portal" required="true"/>
+        <property name="wpProfileLocation" label="Portal Profile Home" description="The location of the portal war file." category="Portal" required="false"/>         
```

Comment:
Introducing portalHost property breaks the plugins backward compatibility because it is required.  The portalHost and wpProfileLocation is actually already defined at the container level and should not be defined on the deployed.  From the backing script, these properties can be accessed via the deployed like deployed.container.cell.portalHost.